hadoop-hdds/common/src/main/java/org/apache/hadoop/hdds/cli/GenericCli.java

```diff
@@ -70,19 +75,33 @@ protected void printError(Throwable error) {
     } else {
       System.err.println(error.getMessage().split("\n")[0]);
     }
-    if(error instanceof MissingSubcommandException){
-      System.err.println(((MissingSubcommandException) error).getUsage());
-    }
   }
 
   @Override
   public Void call() throws Exception {
-    throw new MissingSubcommandException(cmd.getUsageMessage());
```

Comment:
LIKE. Thanks to fix this part (and the similar items). It's better to use the included CommandLine.ParameterException...